cognitoToDDB.js

```diff
@@ -41,44 +41,42 @@ var ddb = new aws.DynamoDB({apiVersion: '2012-10-08'});
 exports.handler = async (event, context) => {
     console.log(event);
 
-    let date = new Date();
+    const date = new Date();
 
     const tableName = process.env.TABLE_NAME;
     const region = process.env.REGION;
 
-    console.log("table=" + tableName + " -- region=" + region);
+    console.log(`table=${tableName} -- region=${region}`)
 
-    aws.config.update({region: region});
+    aws.config.update({ region });
+    
+    if (!event.request.userAttributes.sub) {
+        // Nothing to do, the user's email ID is unknown
```

Comment:
This is actually the UUID of the user (it's a unique id specific to the user pool). The email is at event.request.userAttributes.email